styles/main.css

```diff
@@ -0,0 +1,1096 @@
+
+b, strong {
+  font-weight: bold; }
+
+dfn {
+  font-style: italic; }
+
+h1 {
+  font-size: 2em;
+  margin: 0.67em 0; }
+
+mark {
+  background: #ff0;
+  color: #000; }
+
+small {
+  font-size: 80%; }
+
+sub {
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline; }
+
+sup {
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline;
+  top: -0.5em; }
+
+sub {
+  bottom: -0.25em; }
+
+img {
+  border: 0; }
+
+svg:not(:root) {
+  overflow: hidden; }
+
+figure {
+  margin: 1em 40px; }
+
+hr {
+  box-sizing: content-box;
+  height: 0; }
+
+pre {
+  overflow: auto; }
+
+code, kbd, pre, samp {
+  font-family: monospace, monospace;
+  font-size: 1em; }
+
+button, input, optgroup, select, textarea {
+  color: inherit;
+  font: inherit;
+  margin: 0; }
+
+button {
+  overflow: visible;
+  text-transform: none; }
+
+select {
+  text-transform: none; }
+
+button, html input[type="button"] {
+  -webkit-appearance: button;
+  cursor: pointer; }
+
+input[type="reset"], input[type="submit"] {
+  -webkit-appearance: button;
+  cursor: pointer; }
+
+button[disabled], html input[disabled] {
+  cursor: default; }
+
+button::-moz-focus-inner {
+  border: 0;
+  padding: 0; }
+
+input {
+  line-height: normal; }
+  input::-moz-focus-inner {
+    border: 0;
+    padding: 0; }
+  input[type="checkbox"], input[type="radio"] {
+    box-sizing: border-box;
+    padding: 0; }
+  input[type="number"]::-webkit-inner-spin-button, input[type="number"]::-webkit-outer-spin-button {
+    height: auto; }
+  input[type="search"] {
+    -webkit-appearance: textfield;
+    box-sizing: content-box; }
+    input[type="search"]::-webkit-search-cancel-button, input[type="search"]::-webkit-search-decoration {
+      -webkit-appearance: none; }
+
+fieldset {
+  border: 1px solid #c0c0c0;
+  margin: 0 2px;
+  padding: 0.35em 0.625em 0.75em; }
+
+legend {
+  border: 0;
+  padding: 0; }
+
+textarea {
+  overflow: auto; }
+
+optgroup {
+  font-weight: bold; }
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0; }
+
+td, th {
+  padding: 0; }
+
+body {
+  font-family: 'Lato', sans-serif; }
+  body * {
+    box-sizing: border-box; }
+
+.structure {
+  max-width: 1200px;
+  margin: 0 auto;
+  position: relative;
+  padding: 0 15px; }
+
+.page-header .structure,
+.page-footer .structure {
+  padding: 60px 15px; }
+
+.page-header {
+  background: -webkit-linear-gradient(top, #373737, #212121);
+  background: linear-gradient(to bottom, #373737, #212121); }
+
+.title,
+.tagline {
+  color: #fff;
+  text-shadow: #111 0px 0px 10px; }
+
+.title {
+  font-size: 42px;
+  font-weight: 700; }
+
+.tagline {
+  font-size: 24px;
+  font-weight: 300; }
+
+.main-content-wrap {
+  background: #eee;
+  border-top: 1px solid #111;
+  border-bottom: 1px solid #111; }
+
+.main-content {
+  padding-top: 40px; }
+
+.page-footer {
+  background: #212121;
+  color: #fff;
+  margin-top: 100px; }
+
+/*# sourceMappingURL=main.css.map */
```

Comment:
again I wasn't sure if this should be committed, you're probably right that it shouldn't